mingw-w64-openmp/PKGBUILD

```
@@ -0,0 +1,90 @@
+# Contributor: Mehdi Chinoune <mehdi.chinoune@hotmail.com>
+
+if [[ $MINGW_PACKAGE_PREFIX == *-clang-* ]]; then
+  _clangprefix=1
+fi
+
+_realname=openmp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=12.0.1
+pkgrel=6
+pkgdesc="LLVM OpenMP Library (mingw-w64)"
+url="https://openmp.llvm.org/"
+arch=(any)
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=("custom:Apache 2.0 with LLVM Exception")
+depends=($( (( _clangprefix )) || echo "${MINGW_PACKAGE_PREFIX}-gcc-libs"))
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-clang"
+             $([[ "$CARCH" == "i686" || "$CARCH" == "x86_64" ]] && echo \
+               "${MINGW_PACKAGE_PREFIX}-uasm"))
```

```suggestion
             "${MINGW_PACKAGE_PREFIX}-uasm")
```
Since the whole package can be avoided on clangarm64